### ISSUE TITLE
[ENHANCEMENT]  Add locations screen

### ISSUE THREAD

**Issue by mrchrisadams** [thegreenwebfoundation/admin-portal#375]
**Is your feature request related to a problem? Please describe.**

We want to represent that a given hosting provider can be active in more than one location. Rather than jamming it into an existing form, we'd like to see if this works better as a separate step in the form wizard we are building.

**Describe the solution you'd like**

We would add a new step in the multi step form where users would list at least one location that their organisation is active in.

**Implementation**

We'd need to extend the ProviderRegistrationView with a something like a new entry in the FORMS list.

So here,

```py
FORMS = [
        (Steps.ORG_DETAILS.value, OrgDetailsForm),
        (Steps.SERVICES.value, ServicesForm),
        (Steps.GREEN_EVIDENCE.value, GreenEvidenceForm),
        (Steps.NETWORK_FOOTPRINT.value, NetworkFootprintForm),
        (Steps.CONSENT.value, ConsentForm),
    ]
```
We'd have something like this, where we create a new OrgLocationForm that contains the relevant convenient formset

```py
FORMS = [
        (Steps.ORG_DETAILS.value, OrgDetailsForm),
        (Steps.LOCATIONS.value, OrgLocationForm),
        (Steps.SERVICES.value, ServicesForm),
        (Steps.GREEN_EVIDENCE.value, GreenEvidenceForm),
        (Steps.NETWORK_FOOTPRINT.value, NetworkFootprintForm),
        (Steps.CONSENT.value, ConsentForm),
    ]
```

And update the templates accordingly:

```py
TEMPLATES = {
        Steps.ORG_DETAILS.value: "provider_registration/about_org.html",
        Steps.LOCATIONS.value: "provider_registration/locations.html",
        Steps.SERVICES.value: "provider_registration/services.html",
        Steps.GREEN_EVIDENCE.value: "provider_registration/evidence.html",
        Steps.NETWORK_FOOTPRINT.value: "provider_registration/network_footprint.html",
        Steps.CONSENT.value: "provider_registration/consent.html",
    }
```

**Comment by tortila** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Accept multiple locations in the registration form](https://trello.com/c/hybcW5PY/79-accept-multiple-locations-in-the-registration-form)

**Comment by mrchrisadams** [comment]
Closing this as we resolved it in #378 

https://github.com/thegreenwebfoundation/admin-portal/pull/378
